src/plugins/intel_gna/tools/ncc_style_config/gna.style

```diff
@@ -0,0 +1,129 @@
+# custom OpenVINO GNA values
+CppMethod: '^(operator\W+|[a-z_\d]+|signaling_NaN|quiet_NaN)$'
+ClassName: '^([A-Z][\w]+|b?float16|numeric_limits|ngraph_error|stopwatch|unsupported_op)$'
+StructName: '^([A-Z][\w]+|element_type_traits|hash|oi_pair)$'
+FunctionName: '^(operator\W+|[a-z_\d]+)|PrintTo$'
+Namespace: '^([a-z\d_]*|InferenceEngine)$'
+NamespaceAlias: '^([a-z\d_]+|InferenceEngine)$'
+UnionName: '[A-Z][\w]+$'
+TemplateTemplateParameter: '[A-Z][\w]+'
+NamespaceReference: '^([a-z\d_]+|InferenceEngine|GPUContextParams)$'
+TemplateNonTypeParameter: '^\w*$'
+ClassTemplate: '^([A-Z][\w]+|element_type_traits)$'
+TemplateTypeParameter: '^\w*$'
+ParameterName: '^\w*$'
+FunctionTemplate: '^(operator.+|[\w]+|Impl<.*>)$'
+TypeAliasName: '^\w+$'
+VariableReference: '^\w+$'
```

Comment:
how this config is different from common one?
can / should we explicitly denote options which are different with OpenVINO default config?

Reply:
For now it is the same just first line is different. Currently I am working on changes for this file locally, but before going more into details I wanted to ensure that it will be possible to use custom style file.
Do you mean to group in separate sections with extra comments options which are different and which align with openvino.style?

Reply:
> Do you mean to group in separate sections with extra comments options which are different and which align with openvino.style?

In ideal case, we need to have exactly the same naming convention across all OV library

But if for some reason you need something different for GNA (maybe some exception like we have in common file), it would be nice to highlight this difference

Reply:
Currently there is no possibility to inherit style file. The gna.style file in this PR is the same as openvino.style file, but will be changed in the future and then all deviation will be grouped and marked.

Reply:
can you provide cases why do you need to deviate?

Reply:
E.g. We would like to use more strict rules for variable names and class field names.

Reply:
I would prefer to use Openvino rules (described in the common style config) and specific plugin rules to avoid asynchronous changes.
@marcinkusm  can the current approach guarantee that  changes in openvino.style will be automatically applied to plugin rules?

Reply:
@mryzhov AFAIK for now, there is not possibility to inherit rules like in e.g. clang-format.